webpack/webpack.common.js

```diff
@@ -323,19 +338,13 @@ module.exports = () => {
                 }),
                 new VueLoaderPlugin(),
             ],
-            resolveLoader: {
-                alias: {
-                    text: 'raw-loader'
-                }
-            },
             resolve: {
                 modules: [Path.resolve(__dirname, PROJECT_RELATIVE_NODE_MODULES_PATH)],
                 alias: {
                     ...javascriptRelativeFilepathToAbsoluteFilepathLookup,
                     ...templateFilepathLookup,
                     ...imageFilepathLookup,
                     ...nodeModulesAliases,
-                    ...parsedPackageJSONFilepaths,
```

Comment:
This is no longer needed, it was to accommodate and early iteration of `declarations.d.ts`